src/sdltiles.cpp

```diff
@@ -3624,6 +3624,38 @@ int projected_window_height()
     return get_option<int>( "TERMINAL_Y" ) * fontheight;
 }
 
+// Measures scaling factor for high-dpi displays
+static std::pair<float, float> get_display_scale( int display_index )
+{
+#if SDL_VERSION_ATLEAST(2,26,0)
+    int x = SDL_WINDOWPOS_CENTERED_DISPLAY( display_index );
+    int y = SDL_WINDOWPOS_CENTERED_DISPLAY( display_index );
```

Comment:
You should silence clang-tidy for this error, it's not relevant here.
```suggestion
    // NOLINTNEXTLINE(cata-combine-locals-into-point)
    int x = SDL_WINDOWPOS_CENTERED_DISPLAY( display_index );
    // NOLINTNEXTLINE(cata-combine-locals-into-point)
    int y = SDL_WINDOWPOS_CENTERED_DISPLAY( display_index );
```